aries-site/src/data/structure.js

```diff
@@ -550,6 +550,7 @@ export const structure = [
       component: () => <HeaderPreview />,
       justify: 'start',
     },
+    relatedContent: ['Button', 'Menu', 'TextInput', 'Dashboards'],
```

Comment:
global sidebar ?

Reply:
search also may help theres some information on search page about accessibility such as `aria-labelledby`

Reply:
Agreed, and it should tie into Matt's navigation pattern work as well once that is merged.